src/assessment.py

```diff
@@ -59,16 +61,18 @@ def post_assessment():
         return e, 400
     except OpenaiServerError as e:
         return e, 503
+    except BedrockServerError as e:
+        return e, 503
     except requests.exceptions.ReadTimeout as e:
-        return f"OpenAI timeout: #{e}: ", 504
+        return f"LLM timeout: #{e}: ", 504
```

Comment:
unfortunately we depend on "openai" being present in this string, here: https://github.com/code-dot-org/code-dot-org/blob/0b263e3f3957bade5036eb8d54f98a06e2bda876/dashboard/app/jobs/evaluate_rubric_job.rb#L193-L207

before merging this PR, please make sure dashboard is updated to accept both strings. matching on "LLM timeout" rather than just "LLM".

Reply:
I updated this to return either openai or bedrock depending on the set llm_model value. I also updated cdo to match on either value in the code you linked in this PR: https://github.com/code-dot-org/code-dot-org/pull/60063